### PR TITLE
[iOS] [#790] Remove MapView's bottom toolbar

### LINKED ISSUE #790: [iOS] Remove MapView's bottom toolbar
**Idea Description**

- Title says all. MapView's bottom toolbar isn't needed. So please remove that.

**Reference images and links**
スクリーンショット 2022-09-28 19 03 32

### PR DESCRIPTION
**Pull request by coffmark**
## Issue
- close #790 

## Overview (Required)
- Remove MapView's bottom toolbar    git push --set-upstream origin feature/remove-toobar-button_#790

## Links
- 

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app-ios/Sources/MapFeature/MapView.swift [hunk 1]
```diff
@@ -43,17 +43,6 @@ public struct MapView: View {
                         .padding(14)
                         .navigationTitle(StringsKt.shared.title_map.localized())
                         .navigationBarTitleDisplayMode(.inline)
-                        .toolbar {
-                            ToolbarItem(placement: .bottomBar) {
-                                Button(
-                                    action: { viewStore.send(.tapPin) },
```
Comment (ry-itto) [inline app-ios/Sources/MapFeature/MapView.swift:49]: Can you remove `tapPin` action as well? 🙏🏼

**Comment by coffmark** [comment]
Sorry. 🙏

**Comment by ry-itto** [comment]
Please fix some conflicts 🙏🏼

**Review by ry-itto — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**